torch/csrc/jit/passes/quantization.cpp

```diff
@@ -709,21 +757,55 @@ void InsertObserversHelper::addIntermediateValuesToSkipObserver(
   }
 }
 
+void InsertObserversHelper::fillBoundaryValueMap(
+    script::Module& module, const std::string& method_name) {
+  auto graph = module.get_method(method_name).graph();
+  std::stack<Block*> blocks_to_visit;
+  blocks_to_visit.push(graph->block());
+  auto* self = graph->inputs()[0];
+  while (!blocks_to_visit.empty()) {
+    Block* b = blocks_to_visit.top();
+    blocks_to_visit.pop();
+    for (Node* n : b->nodes()) {
+      if (n->kind() == prim::CallMethod) {
+        auto m = getInvokedModule(module, n, self);
+        auto g = m.get_method(n->s(attr::name)).graph();
+        // add mapping from callsite value to value in called graph
+        for (auto i = 0; i < g->outputs().size(); ++i) {
+          auto* return_val = g->outputs()[i];
+          boundary_value_map_[n->outputs()[i]].push_back(return_val);
```

Comment:
Shouldn't the graph input/output be used as the key in the map? Same graph can be called from several different places, so the same `return_val` can correspond to different `n->output(i)`, which I assume we want to store in the vector.

Reply:
It's the reverse, this is used when we want to find the qconfig for a Value in caller graph